src/pages/users/UsersPage.tsx

```diff
@@ -153,6 +193,24 @@ const UsersPage: FunctionComponent = () => {
     };
     const onSubmitForm = handleSubmit(onSubmit);
 
+    const handleCellEditingStopped = useCallback(
+        (event: CellEditingStoppedEvent<UserInfos>) => {
+            if (event.valueChanged && event.data) {
+                UserAdminSrv.udpateUser(event.data)
+                    .catch((error) =>
+                        snackError({
+                            messageTxt: error.message,
```

Comment:
Maybe to specify at project level, but I think it's annoying to have HTTP errors in the snack bar

Reply:
yes this is an ongoing topic